src/psyclone/tests/generator_test.py

```diff
@@ -98,7 +98,7 @@ def teardown_function():
 
 def test_non_existant_filename():
     ''' checks that alg_gen raises appropriate error when a
-    non-existant filename is supplied '''
+        non-existant filename is supplied '''
```

Comment:
non-existent

Reply:
I think I've always spelled that incorrectly. I shall try not to from now on and have replaced all occurrences in PSyclone (of which there were quite a few).

Reply:
Thank you for changing them!